azure-pipeline-templates/verbose-tests.yml

```diff
@@ -112,6 +119,25 @@ steps:
   - script: cat ${{ parameters.spn_credential_config }}
     displayName: Print SPN config file
 
+  # Create azurite config file if we need to test it
+  - script: |
+      cd ${{ parameters.working_dir }}
+      ${{ parameters.working_dir }}/blobfuse2 gen-test-config --config-file=azure_key.yaml --container-name=${{ parameters.container }} --temp-path=${{ parameters.temp_dir }} --output-file=${{ parameters.azurite_config }}
+    displayName: Create Key Config File
+    # Note: It is fine to hardcode these credentials since they are publicly documented fake credentials for Azurite
+    env:
+      ACCOUNT_TYPE: ${{ parameters.account_type }}
+      NIGHTLY_STO_ACC_NAME: 'devstoreaccount1'
```

Comment:
lets not checkin this part where account name and key are mentioned in plain text, we can define new parameters in lib and use them instead.

Reply:
oh ok just saw the comment above now,

Reply:
yeah its fine, Azurite also documents them publicly